src/modules/ekf_att_pos_estimator/estimator_22states.cpp

```diff
@@ -2599,7 +2599,7 @@ void AttPosEKF::CovarianceInit()
     P[13][13] = sq(0.2f*dtIMU);
 
     //Wind velocities
-    P[14][14] = 0.0f;
+    P[14][14] = 0.01f;
```

Comment:
This is interesting - did you see a performance improvement / smoother estimate with this?